sdk/js/index.ts

```diff
@@ -167,32 +260,34 @@ export default class StreamManager {
       throw new Error("Error while processing trasactions");
 
     return new Stream(
```

Comment:
if we await for `transaction` to mine, we can return `await Stream.fromEventLog` using `transaction.logs[-1]` (since it should be the last log emitted) and avoid the RPC call to get `streamId` above while adding better devex around handling failures of this transaction type

Reply:
we are moving this to another PR